compiler/ir/ir.tree/gen/org/jetbrains/kotlin/ir/declarations/IrProperty.kt

```diff
@@ -45,7 +55,7 @@ abstract class IrProperty :
 
     override fun <D> transformChildren(transformer: IrElementTransformer<D>, data: D) {
         backingField = backingField?.transform(transformer, data) as? IrField
-        getter = getter?.run { transform(transformer, data) as IrSimpleFunction }
-        setter = setter?.run { transform(transformer, data) as IrSimpleFunction }
+        getter = getter?.transform(transformer, data) as? IrSimpleFunction
```

Comment:
I'm not 100% sure which behavior we want, but restored the old one just in case:
```
getter = getter?.transform(transformer, data) as IrSimpleFunction?
```

Reply:
Well, if it's a bug to return incompatible element from `transform` (I also think it is), then maybe catch it everywhere  (not just in IrProperty)?

Reply:
Yes, indeed! I'll do it separately after merging this PR though.